tests/std/tests/P0912R5_coroutine/test.cpp

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 #include <version> // TRANSITION, P0912R5 Library Support For Coroutines

```

Comment:
Pre-existing: Wat? The test for [feature] has two transition comments suggesting things that need to change after we implement [feature]??!?

Reply:
Ah, the intent is to exclude Clang from this test. Could we please change both comments (or remove one and change the other) to `TRANSITION, Clang 14`?

Reply:
We should also add a `/std:c++20` config to the `env.lst` - it seems I overlooked this one implementing GH-1814. Maybe line 15?